commerce-inventory-service/src/main/resources/META-INF/custom-sql/default.xml

```diff
@@ -88,4 +88,28 @@
 				(CIWarehouseItem.sku = ?)
 		]]>
 	</sql>
+	<sql id="com.liferay.commerce.inventory.service.persistence.CommerceInventoryWarehouseItemFinder.countUpdatedItemsByC_M">
+		<![CDATA[
+			SELECT
+				COUNT(1) AS COUNT_VALUE
```

Comment:
Everywhere I see `COUNT` queries in portal custom SQL they use `COUNT(DISTINCT [primary key])`. I'm not sure why this is.... but it could be due to strange behavior in certain DBMSs. Not sure whether we need to conform to that pattern here, but it might be a good idea. Something to think about.